pages/common/tb.md

```diff
@@ -0,0 +1,27 @@
+# tb
+
+> Effectively manage your tasks and notes across multiple boards from within your terminal.
```

Comment:
We try to avoid using the 2nd person in our pages. Let us rephrase this without using "your"